steward/src/commands/cork_proposal/aave_v2_stablecoin/sweep.rs

```diff
@@ -0,0 +1,68 @@
+use crate::{application::APP, cellars, commands::cork_proposal::print_proposal, prelude::*};
+use abscissa_core::{clap::Parser, Command, Runnable};
+use ethers::types::Address;
+use steward_proto::steward::{
+    aave_v2_stablecoin_governance::{Function, Sweep},
+    governance_call::Call,
+    AaveV2StablecoinGovernance, GovernanceCall,
+};
+
+/// Sweep subcommand
+#[derive(Command, Debug, Parser)]
+#[clap(
+    long_about = "DESCRIPTION\n\nSchedule sweep Cosmos address of target cellar when chain reaches specified height"
+)]
+pub struct SweepCmd {
+    /// Token address to transfer out of Cellar.
+    #[clap(short = 't', long)]
+    token: Address,
+
+    /// Address to sweep token into
+    #[clap(short = 'd', long)]
+    destination_address: Address,
+
+    /// Target contract for scheduled cork.
+    #[clap(short, long)]
+    cellar_id: String,
+
+    /// Block height to schedule cork.
+    #[clap(short, long)]
+    block_height: u64,
+
+    /// Only print JSON output, omitting explanatory text
+    #[clap(short, long)]
+    quiet: bool,
+}
+
+impl Runnable for SweepCmd {
+    fn run(&self) {
+        abscissa_tokio::run_with_actix(&APP, async {
+            cellars::validate_cellar_id(&self.cellar_id)
+                .await
+                .unwrap_or_else(|err| {
+                    status_err!("cellar ID validation error:: {}", err);
+                    std::process::exit(1);
+                });
+
+            let governance_call = GovernanceCall {
+                call: Some(Call::AaveV2Stablecoin(AaveV2StablecoinGovernance {
+                    function: Some(Function::Sweep(Sweep {
+                        token: format!("{:#x}", self.token),
```

Comment:
`ethers` has issues with printing addresses, I assume you noticed their weird truncation and that this implementation works? Maybe this is better than the jank-ass way it's done in the orchestrator atm: https://github.com/PeggyJV/gravity-bridge/blob/4a107a812e4c5afd10ba795b016f6558b206e5f8/orchestrator/gravity_utils/src/ethereum.rs#L39-L41

Reply:
Yeah this was the solution I found on stack overflow 😅